en/advanced_config/advanced_mc_position_tuning.md

```diff
@@ -1,96 +1,103 @@
+
+<span></span>
+> **Tip** Follow the instructions in the [Multicopter PID Tuning Guide](https://docs.px4.io/en/advanced_config/pid_tuning_guide_multicopter.html) *before* doing any of the higher-level related control tuning described here. Do not use the advanced position control tuning parameters to fix bad tracking or vibration!
+
+## Overview
 
-## Short Background
-The position controller consists of an outer P-position control loop and an inner PID-velocity control loop. Depending on the control mode either
-the velocity control loop only is active or both. For the remaining part of this tutorial the term **position-control** represents the position and velocity control loop
-activated and **velocity-control** when just the velocity control loop is in use.
+Higher-level tuning applies to any parameter that has an effect on the desired position/velocity setpoint, in contrast to [lower-level tuning](pid_tuning_guide_multicopter.md) that influences the tracking.
 
-## Precondition
-Before doing any higher-level related control tuning, please follow first the instruction [PID-tuning](https://docs.px4.io/en/advanced_config/pid_tuning_guide_multicopter.html).
-It is not recommended to use the advanced position control tuning parameters as a fix for bad tracking or vibration.
+The set of higher-level tuning parameters can be split into two sections: tuning parameters for [Manual/Stabilized](#manual_mode) mode and tuning parameters for [Mission](#mission_mode) mode. Some parameters will have an effect on both modes. 
 
-## Manual Position Control Mode
-In manual control mode the stick inputs are mapped either to **position-control** or **velocity-control**. Position-control is active when the stick inputs are within 
-the deadzone [MPC_HOLD_DZ](../advanced_config/parameter_reference.md#MPC_HOLD_DZ) and velocity control otherwise. All the parameters below are tuning parameters and cannot
-be mapped directly to the physical quantity. 
+> **Note** The position controller ([diagram here](https://dev.px4.io/en/flight_stack/controller_diagrams.html#multicopter-position-controller)) consists of an outer **P** position-control loop and an inner **PID** velocity-control loop. 
+Depending on the control (flight) mode either both loops are active or just the velocity control loop. 
+For the remainder of this topic the term **position-control** represents the case where both loops are active while **velocity-control** refers to the case when only the velocity control loop is in use.
+
+## Manual/Stabilized Mode {#manual_mode}
+
+In [Manual/Stabilized](../flight_modes/manual_stabilized.md) mode the stick inputs are mapped either to **position-control** or **velocity-control**.
+ 
+Position-control is active when the stick inputs are within the deadzone [MPC_HOLD_DZ](../advanced_config/parameter_reference.md#MPC_HOLD_DZ), and velocity-control otherwise. All the parameters below are tuning parameters and cannot be mapped directly to the physical quantity.
```

Comment:
The position-control and velocity-control terminology are a bit annoying in the context of describing flight modes, because we tend to think of position control as being a mode that controls the position of the vehicle - like altitude mode ... and manual mode is "not position controlled" in this context - it can drift with wind. 

I understand that you're saying that in the deadzone both the velocity and position cotnrol loops are active, while outside the zone just the velocity loop is active (and I guess that is because inside we're using the position loop to level attitude?) 

But it would be great if we could use a different terminology that didn't clash.

Reply:
Ok, but where is the definition? To me the naming are already messed up anyway, and also not correct in QGC.  I will quickly try to leverage everything and maybe the naming in general is something to review. There are terminology that I cannot change, even if a mode has a name that has a similar name as the control loop.
I first write down the mode (how it is called in PX4). In parenthesis I write down the stage at which the control pipeline is entered. 
The entire control-pipeline is:
position-velocity-attitude-rate-control (only P/PID loops are considered): 

If I write position control, then the entire pipeline is used. If I write velocity control, then the pipeline from velocity up to rate control is active. Attitude control means that attitude + rate is active, and rate refers to the control part where only rate is active. 

Manual (everything that has RC stick inputs)
-- Acro (rate-control)
-- Rattitude (attitude + rate control): depending on stick (deadzone threshold) attitude or just rate control is active
-- Stabilized (throttle + attitude-control: throttle + roll/pitch/yawt)
-- Altitude (position/velocity and attitude control: Altitude is controlled using position/velocity control                    depending on stick (deadzone threshold); roll/pitch/yaw used for xy
-- Position (position/velocity control in 3D: depending on stick (deadzone threshold) either position control or just velocity is active).

Auto:
-- Mission/RTL/Reposition and so on: they all use position control

Offboard:
Each stage of the control pipeline can be accessed trough offboard 

It is important to note that I cannot call position/velocity/atttiude/rate-control differently, because that is the correct name and represents the kind of control. In contrast, a mode name is something that we made up and has different naming convention based on the autopilot.

Reply:
Thanks for taking the time to put that explanation together. I will think about it tomorrow. If the naming convention is correct then I will need to find another way around this or live with it.